src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  function handleTabClick(tab) {
+    if (selectedTab.title === tab.title) {
+      return null;
+    }
+
+    return onTabSelected(tab);
```

Comment:
no needs to `return` something from this function
```suggestion
    if (selectedTab.title !== tab.title) {
      onTabSelected(tab)
    }
```